justfile

```diff
@@ -57,6 +57,26 @@ clean:
     flutter clean
     cd native && cargo clean
 
+wipe:
+    #!/usr/bin/env bash
+    set -euxo pipefail
+    docker-compose down -v
+    rm -rf data/coordinator/regtest
+    git checkout data/coordinator
+    # Array of possible app data directories (OS dependent)
+    # TODO: Add Linux locations
+    directories=( "$HOME/Library/Containers/finance.get10101.app/Data/Library/Application Support/finance.get10101.app")
```

Comment:
🤔 This will also remove a directory which is not regtest, correct?

Reply:
`git checkout` will restore the seed (it's unfortunately in the regtest dir, the other way would be moving it out of the dir that gets nuked).

the app data will get removed for all, I don't think you can distinguish them in any way. (I would say if we want to do testnet/mainnet, then we should really stick to mobile anyway!).